Runtime/SkyboxAI.cs

```diff
@@ -1,78 +1,142 @@
+            get => _skyboxStyleId;
+            internal set => _skyboxStyleId = value;
         }
 
         [SerializeField]
-        private string skybox_style_name;
+        private string _skyboxStyleName;
 
         [JsonProperty("skybox_style_name")]
         public string SkyboxStyleName
         {
-            get => skybox_style_name;
-            internal set => skybox_style_name = value;
+            get => _skyboxStyleName;
+            internal set => _skyboxStyleName = value;
         }
 
         [SerializeField]
-        private Status status;
+        private Status _status;
 
         [JsonProperty("status")]
         public Status Status
         {
-            get => status;
-            internal set => status = value;
+            get => _status;
+            internal set => _status = value;
         }
 
         [SerializeField]
-        private string type;
+        private string _type;
 
         [JsonProperty("type")]
         public string Type
         {
-            get => type;
-            internal set => type = value;
+            get => _type;
+            internal set => _type = value;
+        }
+
+        [SerializeField]
+        private Material _depthMaterial;
+
+        public Material DepthMaterial
+        {
+            get => _depthMaterial;
+            internal set => _depthMaterial = value;
         }
 
+        [SerializeField]
+        private Material _skyboxMaterial;
+
+        public Material SkyboxMaterial
+        {
+            get => _skyboxMaterial;
+            internal set => _skyboxMaterial = value;
+        }
+
+        [SerializeField]
+        private Texture2D _depthTexture;
+
+        public Texture2D DepthTexture
+        {
+            get => _depthTexture;
+            internal set => _depthTexture = value;
+        }
+
+        [SerializeField]
+        private Cubemap _skyboxTexture;
+
+        public Cubemap SkyboxTexture
+        {
+            get => _skyboxTexture;
+            internal set => _skyboxTexture = value;
+        }
+
+#if UNITY_HDRP
+        [SerializeField]
+        private VolumeProfile _volumeProfile;
+
+        public VolumeProfile VolumeProfile
+        {
+            get => _volumeProfile;
+            internal set => _volumeProfile = value;
+        }
+#endif // UNITY_HDRP
+
         internal void SetMetadata(ImagineResult result)
         {
-            id = result.id;
-            obfuscated_id = result.obfuscated_id;
-            skybox_style_id = result.skybox_style_id;
-            skybox_style_name = result.skybox_style_name;
-            status = result.status;
-            type = result.type;
+            _id = result.id;
+            _obfuscatedId = result.obfuscated_id;
+            _skyboxStyleId = result.skybox_style_id;
+            _skyboxStyleName = result.skybox_style_name;
+            _status = result.status;
+            _type = result.type;
+        }
+
+        internal ImagineResult GetMetadata()
```

Comment:
do we still need this?

Reply:
The idea was if we wanted to convert back to a ImagineResult (I would prefer as implicit operator, but that class is not public yet).